integrations/maven-bloop/src/main/scala/bloop/integrations/maven/MojoImplementation.scala

```diff
@@ -122,7 +120,14 @@ object MojoImplementation {
     val project = mojo.getProject()
     val dependencies =
       session.getProjectDependencyGraph.getUpstreamProjects(project, true).asScala.toList
-    val dependencyNames = dependencies.map(_.getArtifactId()).toList
+    val dependencyNames = dependencies.flatMap(dep => {
+      val matchingArtifacts = project.getArtifacts.asScala.filter(a =>
+        ArtifactUtils.versionlessKey(dep.getArtifact) == ArtifactUtils.versionlessKey(a)
+      )
+      matchingArtifacts.collect {
+        case artifact if artifact.getType == "test-jar" => dep.getArtifactId + "-test"
+      }.toList :+ dep.getArtifactId
```

Comment:
Could we use method instead of ot alias? I find colon operators very confusing.
```suggestion
      }.toList.appended(dep.getArtifactId)
```